README.md

```diff
@@ -116,9 +116,12 @@ _options:_
 * -i    file path to the XLSX for use with xlsx2json
 * -D    (optional) path to the sheets folder; if no folder is specified the the current working directory is used
 
-_[\<sheet name\> ...]:_
+The json2xlsx option take following additional argument:
+* -e    comma delimited list of wildcards patterns of the files which needs to be excluded from processing,
+        i.e. -e UserProfile.json, *-nonprod.json will exclude UserProfile.json and all -nonprod.json files from processing 
```

Comment:
What about wrapping `UserProfile.json, *-nonprod.json` in single quote so that it looks like CLI arg?

Reply:
Wrapped.